Pipfile

```diff
@@ -6,22 +6,20 @@ name = "pypi"
 [packages]
 apispec = {extras = ["yaml"], version = "==5.0"}
 apispec-webframeworks = "==0.5.2"
-flask = "==2.3.3"
-Flask-JWT-Extended = "==4.3"
```

Comment:
It makes me think we should test rest with authentication as well. Do we have an integration test for that ?

Reply:
Let's keep that in mind